{{cookiecutter.repo_name}}/pyproject.toml

```diff
@@ -6,7 +6,7 @@ requires = [
 build-backend = "setuptools.build_meta"
 
 [project]
-name = "{{cookiecutter.repo_name}}"
+name = "{{cookiecutter.package_name}}"
```

Comment:
Hmm -- the docs reckon this name should be what you want the package to be called on PyPI, and allow things like hyphens here. Should this remain just the package name? @orbeckst any opinions here -- what did you have in mind for mdgeomkit?

Reply:
Not sure if my input is still needed but for completeness: I just tried a different name for the *project_name* (the PyPi install name, the one that can have hyphens) from the Python package/module "import" name *package_name* to see if anything would break... and it did:
```
project_name [ProjectName]: mdgeomkit
repo_name [mdgeomkit]: mdgeomkit
package_name [mdgeomkit]: mdgeom
```